docs/tutorials/tut-iiot-cost-estimation.md

```diff
@@ -0,0 +1,148 @@
+
+### IoT Hub message optimization
+Crafting message chunks in an optimal way can lead to large savings. For instance, consider 10 IoT devices that send a 1-KB message every second. These messages will be charged as 10 message chunks per second in total. Consequently, you will need IoT Hub capacity for 10 * 86,400 = 864,000 messages per day, hence at least three S1 units. 
+
+However, if every device instead sends a 4-KB message every 4 seconds, the messages will be charged as 10 message chunks every 4 seconds. This translates to 86,400 / 4 = 21,600 messages per device, for a total of 216,000 messages for all 10 devices per day. In this case, one S1 unit would be enough.
+
+If the requirements state that waiting 4 seconds for the data is not acceptable, that will of course affect the monthly costs. See [IoT Hub quotas and throttling](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-quotas-throttling) and [IoT Hub limits](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#iot-hub-limits) for further relevant information.
+
+### Reference factories
+The following factory setups are provided for reference. A slow node changes every 10 s, a fast one every 1 s.
+
+| Factory size | PLCs | Slow nodes | Fast nodes | Values/s/PLC | VM size | IoT Hub |
+| ------------ | ---: | ---------: | ---------: | -----------: | ------- | ------- |
+| Small	       |   25 |        250 |         50 |           75 | >= B2ms | 8 * S1  |
+| Medium       |   50 |      1,250 |        250 |          375 | >= B2ms | 5 * S2  |
+| Large        |   75 |      3,750 |        750 |        1,125 | >= B4ms | 1 * S3  |
+
+## Simulation
+1. Deploy IoT Hub or [minimal configuration](https://github.com/Azure/Industrial-IoT/blob/master/deploy/templates/azuredeploy.minimum.json) to a resource group called `rg-iiot`, use the reference above to roughly size the hub
+2. To deploy the ARM template on the Azure Portal:
+- Create the resource group and select it
+- Click on *Add* and select *Template deployment (deploy using custom templates)*
+- Click on *Build your own template in the editor*
+- Click on *Load file*, select the template and click on *Save*
+3. On the IoT Hub's blade, add an IoT Edge device
+4. Click on the new IoT Edge device, then *Set Modules -> Add Marketplace Module* and add the following:
+  - OPC Twin
+  - OPC Publisher
+5. Set the container options for the OPC Publisher. The `si` value sets publishing of messages to the IoT Hub to every 10 s. The option `mm` sets the messaging mode and `me` the encoding.   
+```
+{
+    "Hostname": "publisher",
+    "Cmd": [
+        "publisher",
+        "--pf=/appdata/pn.json",
+        "--di=60",
+        "--to",
+        "--aa",
+        "--si=10",
+        "--bs=100",
+        "--fd=true",
+        "--fm=true",
```

Comment:
both --fd and --fm are irrelevant for these tests, --fd is much better to be set to false for load tests, so my recommendation is to remove both flags

Reply:
Removed